src/Network/Email/Email.php

```diff
@@ -1241,6 +1241,16 @@ public static function configTransport($key, $config = null)
     }
 
     /**
+     * Returns an array containing the named transport configurations
+     *
+     * @return array Array of configurations.
+     */
+    public static function configuredTransport()
```

Comment:
If so, then I would argue so is `configTransport()`, which in its most  typical usage in bootstrap.php (provided to nearly every Cake 3 app via the cakephp/app skeleton) takes an array of _multiple_ transports. 

I don't think a "perfect" solution is possible here anymore, but at least without the 's' the new method name can be easily transformed using `str_replace('::config', '::configured', $method)` in exactly the same way that works for everything else using the StaticConfigTrait. See the referenced PR for an example.

Making it 'configuredTransports()' would only further special case it away from the existing pattern and make it harder both for human devs to guess the name based on the setter's name without looking it up in the docs (which itself is _already_ difficult because nothing else consumes two different sets of configs via two different methods) or accessing them programmatically (which is also already hard for the exact same reason, plus no getter even exists yet for the "non-standard" setter method.)

A "proper" solution would probably be to break the Transports out into their own registry that was consumed by Email, like Tasks and Components have for example. Then the setter call would looked something like `EmailTransports::config(Configure::consume('EmailTransports'))`. Ahhh, that's hust satisfying to look at, isn't it? ...but I'm not inclined to wait for Cake 4 for that, personally. ;-)